adapters/repos/db/lsmkv/bucket.go

```diff
@@ -48,6 +48,7 @@ import (
 
 const (
 	FlushAfterDirtyDefault = 60 * time.Second
+	dummyStrategy          = "DUMMY"
```

Comment:
maybe instead of `dummy`, can we use `unknown`? or `not_set`?

Reply:
or `unspecified`?

Reply:
went for `unset`